src/index.js

```diff
@@ -221,6 +228,45 @@ io.on('connection', function(socket) {
     }
   });
 
+  socket.on('cl_buy_life', async (done) => {
+    debug('cl_buy_life');
+
+    if (!user) return done(error('forbidden'));
+    user.spiritPointsCount -= 50;
+    user.lifeUpgrade += 100;
+    try {
+      await User.findByIdAndUpdate(user.id, user);
+      socket.emit(SV_REFRESH_USER, {
+        spiritPointsCount: user.spiritPointsCount,
+        magicPointsCount: user.magicPointsCount,
+        lifeUpgrade: user.lifeUpgrade,
+      });
+      return done(success({spiritPointsCount: user.spiritPointsCount, lifeUpgrade: user.lifeUpgrade}));
+    } catch (e) {
+      return done(error('purchase failed'));
+    }
+  });
+
+  socket.on('cl_all_players', async (done) => {
```

Comment:
Change 'cl_all_players' to 'cl_get_players'.

Reply:
done